addon/mixins/axis-titles.js

```diff
@@ -0,0 +1,174 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+  /**
+   * Toggle axis titles on/off
+   * @type {Boolean}
+   */
+  hasAxisTitles: false,
+
+  /**
+   * Title for the x axis
+   * @type {String}
+   */
+  xValueDisplayName: null,
+
+  /**
+   * Title for the y axis
+   * @type {String}
+   */
+  yValueDisplayName: null,
+
+  /**
+   * Computed title for the x axis, if the `hasAxisTitles` boolean is false
+   * `xAxisTitleDisplayValue` should be an empty string
+   * @type {String}
+   */
+  xAxisTitleDisplayValue: Ember.computed('hasAxisTitles', 'xValueDisplayName', function(){
+    return this.get('hasAxisTitles') ? this.get('xValueDisplayName') : '';
+  }),
+
+  /**
+   * Computed title for the x axis, if the `hasAxisTitles` boolean is false
+   * `yAxisTitleDisplayValue` should be an empty string
+   * @type {String}
+   */
+  yAxisTitleDisplayValue: Ember.computed('hasAxisTitles', 'yValueDisplayName', function(){
+    return this.get('hasAxisTitles') ? this.get('yValueDisplayName') : '';
+  }),
+
+  /**
+   * Computed left margin, if the `hasAxisTitles` is false this should be 0
+   * @type {Number}
+   */
+  horizontalMarginLeft: Ember.computed('hasAxisTitles', function(){
+    return this.get('hasAxisTitles') ? 20 : 0;
```

Comment:
In this case yes, this actually just replaces the value from the no margin mixin, which conflicts with this a bit. I'm moving this to be a computed property on the marginLeft property rather than here so you can pass in a value to horizontalMarginLeft rather than it just being 20.